test/fixtures/mock-contract.ts

```diff
@@ -4,6 +4,11 @@ import { ethers } from 'ethers';
 import { encodeDapiName } from '../../src/utils';
 import { type DeepPartial, encodeBeaconDetails } from '../utils';
 
+const contractFunction = Object.assign(jest.fn(), {
```

Comment:
nit: I actually liked using `.send` as well instead of fn call. But I don't mind this either.